packages/auth/src/grant/middleware.ts

```diff
@@ -0,0 +1,21 @@
+import { ContinueContext } from './routes'
+
+export async function grantLastContinueAttemptMiddleware(
```

Comment:
Should this always be called, regardless of result of `grantRoutes.continue` function? Can you get stuck in a situation where you keep trying to continue, extending `lastContinuedAt` and never being able to successfully continue it?

Reply:
I took a closer look at the spec, and I think I now understand that a `wait` time should be anchored to the last time the "continue" response is given, so I'll change it to reflect that instead.